mm2src/mm2_db/src/indexed_db/indexed_db.rs

```diff
@@ -65,10 +65,15 @@ pub trait TableSignature: DeserializeOwned + Serialize + 'static {
     fn on_upgrade_needed(upgrader: &DbUpgrader, old_version: u32, new_version: u32) -> OnUpgradeResult<()>;
 }
 
+/// A trait representing the operations essential for initializing an IndexedDb instance.
 #[async_trait]
 pub trait DbInstance: Sized {
+    /// Returns the static name of the database.
     fn db_name() -> &'static str;
```

Comment:
`const DB_NAME: &'static str;` should better fit in the trait instead of `fn db_name() -> &'static str;`, not a blocker for this PR tho.

Reply:
yeah, agree, looks better. I can do the refactor in the current PR.
Actually I did it locally, not so many changes, If you wana I can push it.
![image](https://github.com/KomodoPlatform/komodo-defi-framework/assets/52405288/585db3c2-3fc8-4cf2-a709-50685cf458a4)

Reply:
Would be great if you can.

Reply:
Done